RInAppMessaging/Classes/Utilities/AtomicWrapper.swift

```diff
@@ -0,0 +1,29 @@
+/// This wrapper ensures synchronized access to the value only for getter and setter.
+/// Mutating functions, subscript, incrementation etc. are not synchronized.
+@propertyWrapper
+struct AtomicGetSet<Value> {
```

Comment:
Cool! 👍

Reply:
I tried to write some tests but I couldn't produce anything sensible

Reply:
I tried to write some tests but I couldn't produce anything sensible